index.html

```diff
@@ -1,32 +1,47 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Tribute page</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
+<html lang="zxx">
 
-  <body>
-    <main id="main">
-      <h1 id="title">Dr. Makarios Mwape</h1>
-      <div id="img-div"><img src="img/dr_Makarios.jpg" alt="" id="image">
```

Comment:
I think it would be better if you use a section tag instead of a div tag and kindly use a more descriptive class name instead of "id=img-div".😊